### PR TITLE
WIP Add checklist issue number to config

### LINKED ISSUE #109: Add checklist issue number to config.json
Now that we've added a "Help Launch" section to upcoming languages, it'd be really nice to link directly to the checklist issue for the course instead of linking with search params [(source)](https://github.com/exercism/exercism.io/blob/master/x/docs/md/track/LAUNCH.md).

We can add the checklist issue number to the config.json and safely default it to 1 since @kytrinyx posts the issue via the API as soon as a language repo is created.

### PR DESCRIPTION
**Pull request by joeltaylor**
Fixes #109

Reads checklist_issue from `config.json` while assuming that the default issue number is "1"

### CODE REVIEW

**Comment by joeltaylor** [comment]
Took a quick stab at this, but still need to locate where the `config.json` is created for each track and add the `checklist_issue` with `"1"` as the default.

File: test/fixtures/tracks/fake/config.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "slug": "fake",
   "language": "Fake",
   "repository": "https://github.com/exercism/xfake",
+  "checklist_issue": "1",
```
Comment (kytrinyx) [inline test/fixtures/tracks/fake/config.json]: Let's put something not-1 here, and then have a different fixture track without this, which we can default to 1.

**Comment by kytrinyx** [comment]
There are tests for this. They're probably approvals - https://github.com/kytrinyx/approvals

File: test/fixtures/tracks/fake/config.json (hunk 1 above)
Comment (joeltaylor) [inline test/fixtures/tracks/fake/config.json]: Good call, that's be a god way to cover the two scenarios.

Comment (kytrinyx) [inline test/fixtures/tracks/fake/config.json]: Oh, also I think this should be a number, not a string, unless you have a good reason why it shouldn't be.

Comment (joeltaylor) [inline test/fixtures/tracks/fake/config.json]: I felt really dirty adding that as a String. Mostly, I was anticipating a conversion error when adding the repo url with the checklist number....but just realized that's what interpolation is for 😅

Comment (kytrinyx) [inline test/fixtures/tracks/fake/config.json]: Haha, yeah, we can totally do that :)

**Comment by joeltaylor** [comment]
@kytrinyx Do you think it makes sense to go through and add the `checklist_issue` with the default value of `1` to all the other fixtures? Or were you thinking that if it's missing from the config, we would just return `1`? 

I've seen both approaches in the wild. The first option seems more explicit and self-documenting, but I imagine a short note in the README would make option 2 easy to catch.

**Comment by kytrinyx** [comment]
If it's missing, let's just add 1 by default. Way easier than adding it everywhere.

**Comment by kytrinyx** [comment]
@joeltaylor in the future would you mind adding a comment in the PR when you've made changes that need to be looked at? I rely exclusively on my GitHub notifications to review stuff and I don't get notifications for new commits. (Someone else asked about this and I saw that you'd made the change when I was linking to the issue).

Anyway: Yay, and thank you :heart: :)

**Comment by joeltaylor** [comment]
@kytrinyx most definitely! Wasn't sure how vocal to be, but now I know :) Thanks!

**Comment by kytrinyx** [comment]
I <3 vocal--it's the only thing that helps me keep on top of things.

**Pull request merged**